scout/server/blueprints/cases/templates/cases/actionbar.html

```diff
@@ -278,7 +278,7 @@ <h5>
         <form action="{{ url_for('cases.share', institute_id=institute._id, case_name=case.display_name) }}" method="POST">
           <div class="input-group">
             <select class="form-control" name="collaborator">
-              <option>Select institute</option>
+              <option class="placeholder" selected disabled value="">Select institute</option>
```

Comment:
Very good - that was the cause of it, and your solution should be applied to all selects!